prototype/frameworks/langgraph/Containerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.9-slim
+
+WORKDIR /app
+COPY requirements.txt ./
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY . /app
+
+# Expose the ports for FastAPI (8000) and Streamlit (8501)
+EXPOSE 8000
+EXPOSE 8501
+
+# Run the FastAPI server and Streamlit client
+CMD uvicorn fastapi_server:app --host 0.0.0.0 --port 8000 & \
+    streamlit run streamlit_client.py --server.address=0.0.0.0 --server.port=8501
```

Comment:
We would need a separate `Container` file each for 1) the langgraph agent FastAPI server 2) Streamlit server
And then to deploy it on OpenShift, we would also need to define the application deployment files such as `deployment.yaml`, `service.yaml` etc for both the agent and the streamlit app. I think for the initial experimental prototype we don't necessarily need these. We can maybe include these in a separate PR.